src/sass/home/Member.scss

```diff
@@ -80,7 +80,6 @@
   h4 {
     @extend ._vertical-center;
 
-    top: 30%;
```

Comment:
Just curious what this change is for

Reply:
I'm not sure exactly why, but I think pulling in the `react-bootstrap` scss (see https://github.com/MAPC/metro-mayors-housing-task-force/pull/124/files#diff-d9b406b9ce46e9b898af54cbfbd356ee523ae2578050f31092f1a20bf8f7e4a9R1) made the text on the MuniMember shift down (i.e. made this style unnecessary).